### PR TITLE
Add CWL definition for hazard (post UKCP18 support)

### PR DESCRIPTION
**Pull request by ciaransweet**
# What this PR is

This PR adds `hazard_workflow.cwl` and removes `workflow.cwl` (as this seemed a bit more involved that necessary)

The workflow wraps the `os_climate_hazard` CLI invocation, utilising the newly built: `public.ecr.aws/c9k5s3u3/os-hazard-indicator` image

I've run this locally and it generates indicators as if it was the CLI running locally. You can follow the addition to the README on how to do this.

### CODE REVIEW

**Comment by ciaransweet** [comment]
@j08lue probably worth a look + might be handy if you can give a go running this locally yourself? I can provide the input vars 

Be nice to confirm it's not just a 'works on my machine'

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Copy `hazard_workflow_input_example.yml`, rename to `hazard_workflow_input.yml`,
 You can then invoke the CWL with:
 
 ```
-$ cwltool hazard_workflow.cwl hazard_workflow_input.yml
+$ cwltool hazard_workflow.cwl#produce-indicator hazard_workflow_input.yml
```
Comment (j08lue) [inline README.md]: Should be `#produce-hazard-indicator` - or should be changed in the CWL file.

**Review by j08lue — CHANGES_REQUESTED** [review]
Running the workflow worked fine, but README and CWL do not agree on the graph ID.

File: README.md (hunk 1 above)
Comment (ciaransweet) [reply README.md]: Woops! Fixed.

**Review by j08lue — APPROVED** [review]

**Comment by ciaransweet** [comment]
@j08lue I've made no additional changes, just had to do a fair bit of squashing/rebasing so that we don't fall behind on repo syncs again. Could you just give this one last look? Then any further changes I'll make across both repos as we go.

**Review by j08lue — APPROVED** [review]
I ran the workflow locally with

```
cwltool hazard_workflow.cwl#produce-hazard-indicator hazard_workflow_input.yml
```

and

```
$ cat hazard_workflow_input.yml                                                 
window_years: 20  # type 'int'
store: "./indicator"  # default value of type 'string'.
source_dataset: "UKCP18"  # type 'string'
scenario_list: "[rcp85]"  # type 'string'
indicator: "days_tas_above_indicator"  # default value of type 'string'.
gcm_list: "[ukcp18]"  # type 'string'
central_year_list: "[2020, 2030, 2040]"  # type 'string'
central_year_historical: 2030  # type 'int'
ceda_ftp_username: "user"  # type 'string'
ceda_ftp_url: "ftp.ceda.ac.uk"  # type 'string'
ceda_ftp_password: "pwd"  # type 'string'
```

and it successfully produces the expected JSON and Zarr output.

image

**Pull request merged**